### DISCUSSION TITLE
Changing where Github Desktop copies files for posting to Repository

### DISCUSSION THREAD

**Discussion by ProgramViBee** [community/community#78566] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When creating my repository, and linking it via Github Desktop, I think I accidentally set it to `ProjectName/Subfolder/` instead of `ProjectName/`, and now it is only copying files from `ProjectName/Subfolder/`. I want to fix it so that it will cover Everything in the `ProjectName/` folder.

How can I fix this?

**Comment by erickdc7** [comment]
To fix the issue with your GitHub Desktop repository path, you can follow these steps:

1. **Close GitHub Desktop**: Ensure that GitHub Desktop and any other applications that might be using files from your current directory are closed¹.

2. **Move the Repository**: Physically move the entire repository folder (including the `.git` directory) to the desired location. This is typically done by cutting and pasting the folder in your file system.

3. **Reopen GitHub Desktop**: After moving the repository, open GitHub Desktop. You will likely encounter a "repository not found" error with a blue (!) icon.

4. **Locate the Repository**: Click on the blue (!) icon, and a dialog will open. Click on "Locate..." and browse to the new directory where you moved your repository.

By following these steps, GitHub Desktop should recognize the new location of your repository and include all the files from the `ProjectName/` folder in future operations.

If you encounter any issues during this process, such as GitHub Desktop asking you to create a new repository, make sure that you have moved all the necessary files, including any hidden ones like the `.git` directory. Missing files could cause GitHub Desktop to not recognize the repository correctly.

Remember, it's important to move the entire repository, not just the contents of the `ProjectName/Subfolder/`, to ensure that GitHub Desktop tracks all the files under `ProjectName/`.

I hope this helps you resolve the issue!
